### ISSUE TITLE
Use config param upload_path upon installation

### ISSUE THREAD

**Issue by DanielSchwiperich** [contao/core-bundle#393]
The installation routine creates some empty dirs like the 'files'  folder.

The files folder can be overwritten via the `upload_path` param in the config.yml
`InstallCommand::addEmptyDirs()` is using an array where the entry for the upload path is fixed and not using the config param.

```
    private $emptyDirs = [
        'files',
        'system',
        'templates',
        'web/system',
    ];
``` 

I think `InstallCommand::addEmptyDirs()` should merge in the param from configuration class and drop it from the fixed array definition.

**Comment by aschempp** [comment]
:+1: 

Applies to the `files` path only though because the other folders are only there for backwards compatibility and their name is unchangeable.

**Comment by leofeyer** [comment]
Closed in favor of #394.
